tests/components/unifi/test_switch.py

```diff
@@ -864,7 +874,7 @@ async def test_restore_client_succeed(hass, aioclient_mock):
 
     registry = er.async_get(hass)
     registry.async_get_or_create(
```

Comment:
I noticed `async_get_or_create` positional args are `domain` and `platform` respectively. In this case, `platform` could not be `UNIFI_DOMAIN`.